test/query_test.rb

```diff
@@ -11,4 +11,15 @@ def test_basic
     assert_equal ["Apple", "Milk"], query.execute.map(&:name).sort
   end
 
+  def test_with_effective_min_score
+    store_names ["Milk", "Milk2"]
+
+    assert_equal ["Milk"], Product.search("Milk", min_score: 0.1).map(&:name)
```

Comment:
@ankane I thought this test and the following one shows "how these options affect the search results"? cc @st0012